jupyter_dashboards/nbextension/notebook/dashboard-view/dashboard-view.js

```diff
@@ -69,6 +69,14 @@ define([
     }
 
     PolymerSupport.init();
+    
+    // register delegate handlers that persist across dashboard mode changes  
+    $('#notebook_panel').on('change', '.show-code-label input', function() {
+        $('#notebook-container').toggleClass('hide-code', !this.checked);
+    });
+    $('#notebook_panel').on('click', '.more-info-link', function() {
+        $(this).parents('.help-area').toggleClass('show-more', 200);
+    });
```

Comment:
Well, no, you need to know exactly the same (which elements have the handlers to unregister), the code is two places (register in template vs unregister in JS), and it has to register and re-register every time the template is replaced (vs once in the JS).